### ISSUE TITLE
Keyboard navigation should be available to get into the Compare Versions drop down menu

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-code-manager#1304]
Followup for https://github.com/mozilla/addons-code-manager/issues/757

STR:
1. Load https://code.addons-dev.allizom.org/en-US/browse/618459/versions/1689844/
2. Use keyboard navigation (Tab) or the downwards arrow to get to `Compare Versions` button 
3. Press `Enter` then try to go using the keyboard to `Old version` 
4. Observe which is the next element highlighted  

Expected results:
`Old version` from the the drop down menu is highlighted.

Actual result:
`Comment Summary` button is selected. 

Notes:
If I open the menu and then I click inside it the keyboard navigation can be done.

![tab key](https://user-images.githubusercontent.com/33448286/70129544-9be39200-1687-11ea-9d0c-b297aa7f32ab.gif)
